### PR TITLE
Revisiting art of a graceful exit lesson

### PR DESCRIPTION
**Pull request by zoefarrell**

### CODE REVIEW

File: module6/lessons/Week2/RevisitingTheGracefulExit.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+#### Pivot
+Response: “I’m not familiar with that concept, but it reminds me of ____ . Could I talk a bit about that instead?”
+
+What qualities is this interviewee displaying?
+
+## Students Demonstrate Graceless Interview Answers
+
+You're going to practice graceful exits in a few minutes, but first, let's practice graceless exits! This way you will have a chance to see what it feels like as an interviewer to receive a graceless answer.
+
+Here's how this works, your instructor will start by drawing a random student for the first interviewee. Then they will ask a technical interview question and the first student will give a graceless response. Then that first student becomes the interviewer. The instructor will draw a second random student. The first student will ask a technical interview question and the second student will give a graceless response.
```
Comment (memcmahon) [inline module6/lessons/Week2/RevisitingTheGracefulExit.md:56]: Oo - I love this activity!

File: module6/lessons/Week2/RevisitingTheGracefulExit.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+Today we're going to do our breakout room practice in groups of 3 or 4.
+
+Each person will rotate between being the interviewer, the interviewee, and the observer (for groups of 4 there will be 2 observers).
+
+As the interviewee, answer to the best of your abilities - these questions are especially hard so you will have the opportunity to practice a graceful exit!
+
+After the interviewee answers, the interviewer can give kind and actionable feedback, again referring to the questions above about what additional information the answer tells you about the candidate. The observer is not responsible for asking questions, answering questions, or giving feedback. They can focus on learning from observing how someone else tackles hard questions.
+
+### Questions
```
Comment (memcmahon) [inline module6/lessons/Week2/RevisitingTheGracefulExit.md:83]: yay spicy questions!  I think we probably want to try to get joe to be interviewed as 1 or 2 - I think with his background, 3 and 4 might be less 'required' graceful exit... he might know enough to answer questions in 4, and most of the questions in 3. 🤷‍♀️

**Review by memcmahon — APPROVED** [review]

File: module6/lessons/Week2/RevisitingTheGracefulExit.md (hunk 1 above)
Comment (zoefarrell) [reply module6/lessons/Week2/RevisitingTheGracefulExit.md:56]: Thanks!

File: module6/lessons/Week2/RevisitingTheGracefulExit.md (hunk 2 above)
Comment (zoefarrell) [reply module6/lessons/Week2/RevisitingTheGracefulExit.md:83]: Good call out! I might just make schedules for each group of who is in which role. Also less for them to think about too!

**Pull request merged**